Userland/Utilities/lsusb.cpp

```diff
@@ -18,15 +19,58 @@
 #include <stdio.h>
 #include <unistd.h>
 
+// FIXME: Hmmm, perhaps this should be part of the USBDB library?
+static const StringView s_device_classes[17] = {
```

Comment:
Let's use a more modern `AK::Array` for this :)

Reply:
I'm removing this, we already have support for this in LibUSBDB